### PR TITLE
Storage

### PR DESCRIPTION
**Pull request by potfur**

### CODE REVIEW

File: src/Vox/bootstrap.db.connection.php [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+return array(
+    'container' => array(
+        'storage:connection' => array(
+            'component' => function () {
+                $connectionParams = [
+                    'dbname' => '@database@',
+                    'user' => '@user@',
+                    'password' => '@password@',
+                    'host' => '@host@',
+                    'driver' => 'pdo_mysql',
```
Comment (marcotroisi) [inline src/Vox/bootstrap.db.connection.php:11]: don't forget, we are using SQLite...

File: src/bootstrap.php [hunk 2]
```diff
@@ -1,7 +1,12 @@
 <?php
+$dbConnectionPath = __DIR__ . '/../src/Vox/bootstrap.db.connection.php';
+
 return array(
     'import' => array(
         (array) require __DIR__ . '/../src/Vox/bootstrap.php',
-        (array) require __DIR__ . '/../src/Console/bootstrap.php'
+        (array) require __DIR__ . '/../src/Vox/bootstrap.db.php',
+        is_file($dbConnectionPath) ? (array) require $dbConnectionPath : [],
```
Comment (marcotroisi) [inline src/bootstrap.php:8]: perhaps `file_exists` instead of `is_file`?

**Comment by marcotroisi** [comment]
Great job!

File: src/bootstrap.php (hunk 2 above)
Comment (potfur) [inline src/bootstrap.php:8]: `file_exists` would return `true` also for directories also `is_file` used to be faster

**Pull request merged**